server/@types/forms/index.d.ts

```diff
@@ -15,6 +15,10 @@ declare module 'forms' {
     note?: string
   }
 
+  export interface ReviewGoalForm {
```

Comment:
Basic form type with just an action field - because the "Review Your Goals" page submits the form (POST) to go to the next screen - I've now added an action so we can tell if the user has clicked to Add Another Goal, or submit the form to save the action plan (current behaviour)